### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by mophilly** [enoch3712/ExtractThinker#106] [Q&A] [answered]
New to many things here, yet 30 years in software work. So I am an astute newbie?
I installed package into a pyCharm project. The install seems to be missing parts. E.g. config.py. I don't see a folder for "medium_posts". 
Hints and links warmly accepted.

**Comment by enoch3712** [comment, marked as answer]
Medium posts are going to be removed of the project, they are the legacy use cases, so Config doesnt exist anymore

In terms of documentation you have: https://enoch3712.github.io/ExtractThinker/

Can you run this simple example: https://github.com/enoch3712/ExtractThinker/blob/main/examples/notebooks/basic_example.ipynb

Or this quickstart? https://enoch3712.github.io/ExtractThinker/getting-started/

**Reply by mophilly** [reply]
Thank you. I read the documentation again. I added my API key and was able to run both examples.

**Reply by enoch3712** [reply]
@mophilly 

This comment slipped, sorry! 

Good! No worries! This project is taking off and i will always be here to clarify if needed
